src/rapidjson/allocators.h

```diff
@@ -0,0 +1,271 @@
+// Tencent is pleased to support the open source community by making RapidJSON available.
```

Comment:
I am *Strongly* opposed to directly including this library into the project's source code.

Do proper dependency management with CMake from an official rapidjson release, with appropriate release URL.

Embedding code like this is going to make package maintainers lives *VERY VERY HARD*, and it will literally never be updated if it's directly included in the project's repository.

Reply:
I agree. I was directly including the code because it looks like configure.py still needs to work to pass the CI build. Is that no longer true?

Reply:
You can remove configure.py in your PR.

Can't say it'll be merged though. Not my call.

Reply:
Changed to use CMake for dependency management.